.storybook/DocsDecorator/HealthFactor/SingleDoc/index.styles.ts

```diff
@@ -0,0 +1,61 @@
+import { css, Theme } from '@emotion/react';
+
+const styles = {
+  documentationContainer: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+
+  iconContainer: {
+    display: 'inline-block',
+    boxSizing: 'border-box',
+  },
+
+  linkIconContainer: (theme: Theme) =>
+    css({
+      width: `${theme.spacings.DOUBLE}rem`,
+      height: `${theme.spacings.DOUBLE}rem`,
+      marginLeft: `${theme.spacings.HALF}rem`,
+      color: `${theme.palette.SHADOW}`,
+    }),
+
+  statusIconContainer: {
+    borderRadius: '100%',
+    width: '1.25rem',
+    height: '1.25rem',
+    padding: '0.25rem',
+  },
+
+  positive: (theme: Theme) =>
+    css({ background: '#0A7B0A', color: `${theme.palette.WHITE}` }),
+
+  negative: (theme: Theme) =>
+    css({ background: '#D60A3D', color: `${theme.palette.WHITE}` }),
+
+  missing: (theme: Theme) =>
+    css({ padding: 0, borderRadius: 0, color: `${theme.palette.SHADOW}` }),
+
+  sidebarColumn: {
+    margin: '0.375rem',
+    marginLeft: '0',
+  },
+
+  documentationLink: {
+    margin: '0.25rem',
+  },
+
+  documentationType: {
+    flexGrow: 1,
+  },
+
+  link: (theme: Theme) =>
+    css({
+      textDecoration: 'none',
+      '&:hover': {
```

Comment:
I gave an attempt at the focus, let me know if this is what is needed.

Reply:
Sorry I can't preview and can't tell if that CSS is the same for focus, we are soon going to be building a production version of this @Isabella-Mitchell @LillyHuynh so we might just be able to pull that in when it's ready?

Reply:
👍 Now I should have fixed the preview.
But in case it breaks again when I am off, this is how it looks:
<img width="909" alt="Screenshot 2022-12-15 at 12 21 25" src="https://user-images.githubusercontent.com/90621252/207858099-32940491-caf4-44d6-a4ec-f35a7aeff197.png">

I think you can notice the small overlap. Is this overlap ok or should the content just shift a bit to the bottom?
It is very weird watching the content shifting lower so I preferred an overlap, but let me know what you think.

Reply:
Think I have confused you, appologies. This should be the same as the hover styling with the addition of a double focus ring (to meet new a11y guidelines). Please could you ask @LillyHuynh and @Isabella-Mitchell if you can pull in their code for this when they have done it? FYI @LillyHuynh @Isabella-Mitchell this should be the same as what UX finalise the design for this for an 'inline' link